transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -830,8 +864,8 @@ public void deregister(ChannelPromise promise) {
 
             @Override
             public void beginRead() {
-                EmbeddedUnsafe.this.beginRead();
                 runPendingTasks();
+                EmbeddedUnsafe.this.beginRead();
```

Comment:
@carryxyh why this change ?

Reply:
This is an incorrect modification. Inadvertently changed. Already reverted.